Enrich_methods/material_db_tools.py

```diff
@@ -0,0 +1,444 @@
+from typing import Dict, Union, Optional
+from pyne import material
+from pyne.material import Material, MultiMaterial
+from pyne.material_library import MaterialLibrary
+from pyne import nucname
+from decimal import Decimal, getcontext
+
+
+def make_mat(
+    nucvec: Dict[int, float],
+    density: float,
+    citation: str,
+    molecular_mass: Optional[float] = None,
+    mass_enrichment: Optional[Dict[int, Dict[int, float]]] = None,
```

Comment:
If you set the default as `{}` instead of `None` you can avoid the if statment below and just iterate over a potentially empty dictionary.